Update onchain tx hash type

dolcalmi opened the pull request:
- I am having issues with eslint validations (even with check-code passing). Please validate downloading the branch
- Addresses refactor will be done in another PR https://github.com/GaloyMoney/galoy/pull/656

krtk6160 commented on src/services/notifications/index.ts:
```diff
@@ -19,7 +19,7 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
         logger: logger,
         amount,
-        txid: txId,
+        txid: txHash,
```
I think we can change it, I don't think we have a shared schema of the object that the mobile app expects. @nicolasburtey can confirm

dolcalmi commented:
https://github.com/GaloyMoney/galoy/commit/6f2b6dd4510ac9092cd9b565c875d0e9db608e01
- fixed rebase conflicts
- renamed txId to txHash as suggested by @bodymindarts
